src/components/SpecialOffers/index.js

```diff
@@ -101,7 +101,28 @@ const SpecialOffers = () => {
 
   const [widgets, setWidget] = useState({});
 
-  const loadMore = () => {
+//   const loadMore = () => {
+//     const newWidgets = {
+//         results: [{
+//           "title": "New Clips(1 pack)",
+//           "mrp": "$12.00",
+//           "discounted_price": "$6.00",
+//           "image": "images/of20.png",
+//           "quantity": 4
+//       }],
+//       "moreAvailable": false,
+//     };
+//     setWidget(oldState => {
+//       return {
+//         ...oldState,
+//         results: [...oldState.results, ...newWidgets.results],
+//         moreAvailable: newWidgets.moreAvailable,
+//       }
+//     });
+//   }
+  
+  
+  const load = () => {
```

Comment:
Can we use a better function name here? Also, this function will be declared again and again on every rerender.
Suggestion: Better to use useCallback hook here.